AndroidGesture/app/src/main/java/com/example/ben/androidgesture/PebbleReceiver.java

```diff
@@ -137,4 +143,24 @@ public void onPause() {
         PebbleKit.closeAppOnPebble(getApplicationContext(), PEBBLE_UUID);
     }
 
+    public void saveDataToFile(String x, String y, String z){
+
+        try {
+            Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+            File root = new File(DIRECTORY_PATH);
+            File file = new File(root, "logFile.txt");
+            if(!file.exists()){
+                file.createNewFile();
+            }
+            FileOutputStream fOut = new FileOutputStream(file);
+            OutputStreamWriter outWriter = new OutputStreamWriter(fOut);
+            outWriter.append(x + " " + y + " " + z + " " + timestamp.toString());
```

Comment:
can you add the strings 'X: ' and etc to this line? I want to be able to know which ones the x, y, z without just memorizing the order